release notes/v0.36.0.md

```diff
@@ -0,0 +1,76 @@
+  exec.test.abort("this is the reason");
+}
+
+export function teardown() {
+  console.log("teardown will still be called after test.abort()");
+}
+```
+
+Or during the VU initialization:
+```javascript
+import exec from "k6/execution";
+
+// This won't fail on initial parsing of the script, but on VU initialization.
+if (__VU == 1) {
+  exec.test.abort();
+}
+
+export default function() {}
+```
+
+### k6 inspect extended output ([#2279](https://github.com/grafana/k6/pull/2279))
+
+Following [#2279](https://github.com/grafana/k6/pull/2279), the `k6 inspect` command now supports an `--execution-requirements` flag. When used, the command's output will include fields related to the execution requirements, the configuration k6 derived from the execution context, and will also expose the `maxVUs` and `totalDuration` fields.
+
+
+### Forcing HTTP/1 protocol ([#2222](https://github.com/grafana/k6/pull/2222))
+
+Thanks to the work of [@sjordhani22](https://github.com/sjordhani22), [#2222](https://github.com/grafana/k6/pull/2222) made it possible to force k6 to use version 1.1 of the protocol when firing HTTP requests. 
+
+It can be done by setting the `http2client=0` value in the `GODEBUG` environment variable:
+
+```
+GODEBUG=http2client=0 k6 run testscript.js
+```
+
+**Nota Bene**: the usage of the `GODEBUG` variable is considered temporary, and expected to change in the future. If you start using this feature, keep an eye out for potential future changes.
+
+## Bugs Fixed!
+
+* We had a long-standing issue ([#1771](https://github.com/grafana/k6/issues/1771)) making opening large files in the context of a k6 script somewhat unreliable under specific conditions in the past. This has now been addressed by [#2314](https://github.com/grafana/k6/pull/2314).
+* We addressed an issue uncovered by [our community](https://community.k6.io/t/v0-35-0-grpc-server-reflection-error/2383), which kept our users using GRPC to define multiple services in a single *proto* file. This issue was addressed by [#2265](https://github.com/grafana/k6/pull/2265).
+* Thanks to the contribution of [@Resousse](https://github.com/Resousse), we've now updated k6's [`go-ntlmssp`](https://github.com/Azure/go-ntlmssp) dependency. The updating PR [#2290](https://github.com/grafana/k6/pull/2290) indeed fixes issues with NTLM Authentication backends returning two authorization headers. 
+
+## Maintenance
+
+- `v0.36.0` marks the switch to a new JS module's internal API. Most of our official JS modules have now been ported to this new version, which should make the process of developing modules easier and more streamlined in the future. See for more details: [#2243](https://github.com/grafana/k6/pull/2243), [#2241](https://github.com/grafana/k6/pull/2241), [#2239](https://github.com/grafana/k6/pull/2239), [#2242](https://github.com/grafana/k6/pull/2242), [#2226](https://github.com/grafana/k6/pull/2226), and [#2232](https://github.com/grafana/k6/pull/2232).
```

Comment:
Perhaps, this should be in a separate section :thinking: It is a rather large change, with impact on xk6 extensions. Maybe with some additional details too, e.g. snippet with the new interface. WDYT?

Reply:
You're probably right! As I'm not too familiar with its history, I was hesitant to make it a "breaking change", as I couldn't judge how much breaking it actually was. Also, the documentation doesn't seem to be using the new API: https://k6.io/docs/extensions/guides/create-an-extension/, so I was unsure if we planned to talk about it yet.

cc @na-- what do you think?

Reply:
:+1: for a separate section. Not sure how to format it exactly, since it's not quite a breaking change **yet**, but we plan to deprecate it in the future... :confused: Maybe we can just have a new top-level section of this document called `Extensions`, where we both announce any new official extensions (like [these](https://github.com/grafana/k6/blob/master/release%20notes/v0.35.0.md#new-xk6-extensions)) and also announce k6 changes that affect existing extensions? :thinking:

Reply:
Alright, I'll move it into a dedicated `Extensions` section. Also adding the info that although it is not a breaking change yet, deprecation are to be expected in the future in favor of this new API.

Reply:
I've made changes according to your comments. Could you take a 👀  @yorugac and @na--, and tell me if that works for you 🙇🏻?